BadHabbit/BadHabbit/SceneDelegate.swift

```diff
@@ -33,6 +38,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         
         let window = UIWindow(windowScene: windowScene)
         window.rootViewController = rootTabBarAssembler.assemble()
+        window.backgroundColor = .white
```

Comment:
why ?

Reply:
This line of code was removed, tabBarItems color was changed.

Resolved